app/src/main/java/net/squanchy/eventdetails/EventDetailsService.java

```diff
@@ -44,7 +45,7 @@
                     apiEvent.name,
                     apiEvent.place,
                     apiEvent.experienceLevel,
-                    map(speakers, toSpeakerName())
+                    map(speakers, toSpeakers())
```

Comment:
Thanks for this, we needed to do it at some point, BUT we need to create new view models to include the data we actually need in the screens